pages/mesosphere/dcos/1.14/security/ent/index.md

```diff
@@ -186,16 +186,22 @@ To learn how to do this, see [Controlling user access to services](/mesosphere/d
 
 ## <a name="secrets"></a>Spaces for secrets
 
-The secret path controls which services can access it. If you do not specify a path when storing a secret, any service can access it.
+The secret path controls which services (a service usually is a Marathon application started on DC/OS' Root Marathon)
+can access it. If you do not specify a path when storing a secret, any service can access it.
```

Comment:
I just realize that `If you do not specify a path when storing a secret, any service can access it.` should maybe be replaced while we're here. "If you do not specify a path "is technically wrong. Maybe let's replace it with something in the lines of:

`Note: if you store a secret at the top level then any service can access it!`

and then I think this should go below the table.